## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 445278`
Commit: `a46ae6d698b23f8109907e43cbba26223f84fe6e`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 9 packages marked as broken and skipped:</summary>
  <ul>
    <li>merkaartor</li>
    <li>python312Packages.spyder</li>
    <li>python312Packages.spyder.dist</li>
    <li>python313Packages.spyder</li>
    <li>python313Packages.spyder.dist</li>
    <li>pytrainer</li>
    <li>pytrainer.dist</li>
    <li>spyder</li>
    <li>spyder.dist</li>
  </ul>
</details>
<details>
  <summary>:x: 408 packages failed to build:</summary>
  <ul>
    <li>ants</li>
    <li>apacheHttpdPackages.mod_tile (apacheHttpdPackages_2_4.mod_tile)</li>
    <li>appcsxcad</li>
    <li>c3d</li>
    <li>cloudcompare</li>
    <li>csxcad</li>
    <li>elastix</li>
    <li>elmerfem</li>
    <li>entwine</li>
    <li>exhibit</li>
    <li>expliot</li>
    <li>expliot.dist</li>
    <li>ezminc</li>
    <li>f3d</li>
    <li>f3d.man</li>
    <li>freecad</li>
    <li>froide (python313Packages.froide)</li>
    <li>froide-govplan</li>
    <li>froide-govplan.dist</li>
    <li>froide.dist (python313Packages.froide.dist)</li>
    <li>gdal (python313Packages.gdal)</li>
    <li>gdalMinimal</li>
    <li>gdcm</li>
    <li>gmt</li>
    <li>gplates</li>
    <li>grass</li>
    <li>haskellPackages.hgdal</li>
    <li>haskellPackages.hgdal.doc</li>
    <li>intensity-normalization (python313Packages.intensity-normalization)</li>
    <li>intensity-normalization.dist (python313Packages.intensity-normalization.dist)</li>
    <li>itk (itk_5)</li>
    <li>itk_5_2</li>
    <li>mapcache</li>
    <li>mapnik</li>
    <li>mapproxy</li>
    <li>mapproxy.dist</li>
    <li>mapserver</li>
    <li>mirtk</li>
    <li>mrtrix</li>
    <li>mysql-workbench</li>
    <li>octavePackages.dicom</li>
    <li>octavePackages.mapping</li>
    <li>openems</li>
    <li>openorienteering-mapper</li>
    <li>osmscout-server</li>
    <li>otb</li>
    <li>paraview</li>
    <li>pcl</li>
    <li>pdal</li>
    <li>perl538Packages.Tirex</li>
    <li>perl538Packages.Tirex.devdoc</li>
    <li>perlPackages.Tirex (perl540Packages.Tirex)</li>
    <li>perlPackages.Tirex.devdoc (perl540Packages.Tirex.devdoc)</li>
    <li>postgresql13Packages.postgis</li>
    <li>postgresql13Packages.postgis.doc</li>
    <li>postgresql14Packages.postgis</li>
    <li>postgresql14Packages.postgis.doc</li>
    <li>postgresql15Packages.postgis</li>
    <li>postgresql15Packages.postgis.doc</li>
    <li>postgresql16Packages.postgis</li>
    <li>postgresql16Packages.postgis.doc</li>
    <li>postgresqlPackages.postgis (postgresql17Packages.postgis)</li>
    <li>postgresqlPackages.postgis.doc (postgresql17Packages.postgis.doc)</li>
    <li>postgresql18Packages.postgis</li>
    <li>postgresql18Packages.postgis.doc</li>
    <li>python312Packages.basemap</li>
    <li>python312Packages.basemap.dist</li>
    <li>python312Packages.bsuite</li>
    <li>python312Packages.bsuite.dist</li>
    <li>python312Packages.cartopy</li>
    <li>python312Packages.cartopy.dist</li>
    <li>python312Packages.dcmstack</li>
    <li>python312Packages.dcmstack.dist</li>
    <li>python312Packages.deid</li>
    <li>python312Packages.deid.dist</li>
    <li>python312Packages.dicom-numpy</li>
    <li>python312Packages.dicom-numpy.dist</li>
    <li>python312Packages.dicom2nifti</li>
    <li>python312Packages.dicom2nifti.dist</li>
    <li>python312Packages.dicomweb-client</li>
    <li>python312Packages.dicomweb-client.dist</li>
    <li>python312Packages.dipy</li>
    <li>python312Packages.dipy.dist</li>
    <li>python312Packages.django-bootstrap4</li>
    <li>python312Packages.django-bootstrap4.dist</li>
    <li>python312Packages.django-bootstrap5</li>
    <li>python312Packages.django-bootstrap5.dist</li>
    <li>python312Packages.drf-extra-fields</li>
    <li>python312Packages.drf-extra-fields.dist</li>
    <li>python312Packages.f3d</li>
    <li>python312Packages.f3d.man</li>
    <li>python312Packages.fast-simplification</li>
    <li>python312Packages.fast-simplification.dist</li>
    <li>python312Packages.fiona</li>
    <li>python312Packages.fiona.dist</li>
    <li>python312Packages.folium</li>
    <li>python312Packages.folium.dist</li>
    <li>python312Packages.froide</li>
    <li>python312Packages.froide.dist</li>
    <li>python312Packages.fslpy</li>
    <li>python312Packages.fslpy.dist</li>
    <li>python312Packages.gdal</li>
    <li>python312Packages.gdcm</li>
    <li>python312Packages.geoarrow-pandas</li>
    <li>python312Packages.geoarrow-pandas.dist</li>
    <li>python312Packages.geoarrow-pyarrow</li>
    <li>python312Packages.geoarrow-pyarrow.dist</li>
    <li>python312Packages.geodatasets</li>
    <li>python312Packages.geodatasets.dist</li>
    <li>python312Packages.geopandas</li>
    <li>python312Packages.geopandas.dist</li>
    <li>python312Packages.geoparquet</li>
    <li>python312Packages.geoparquet.dist</li>
    <li>python312Packages.gstools</li>
    <li>python312Packages.gstools.dist</li>
    <li>python312Packages.heudiconv</li>
    <li>python312Packages.heudiconv.dist</li>
    <li>python312Packages.highdicom</li>
    <li>python312Packages.highdicom.dist</li>
    <li>python312Packages.inequality</li>
    <li>python312Packages.inequality.dist</li>
    <li>python312Packages.intensity-normalization</li>
    <li>python312Packages.intensity-normalization.dist</li>
    <li>python312Packages.itk</li>
    <li>python312Packages.libpysal</li>
    <li>python312Packages.libpysal.dist</li>
    <li>python312Packages.lida</li>
    <li>python312Packages.lida.dist</li>
    <li>python312Packages.mapclassify</li>
    <li>python312Packages.mapclassify.dist</li>
    <li>python312Packages.mayavi</li>
    <li>python312Packages.mayavi.dist</li>
    <li>python312Packages.medpy</li>
    <li>python312Packages.medpy.dist</li>
    <li>python312Packages.medvol</li>
    <li>python312Packages.medvol.dist</li>
    <li>python312Packages.momepy</li>
    <li>python312Packages.momepy.dist</li>
    <li>python312Packages.morecantile</li>
    <li>python312Packages.morecantile.dist</li>
    <li>python312Packages.msticpy</li>
    <li>python312Packages.msticpy.dist</li>
    <li>python312Packages.napari-nifti</li>
    <li>python312Packages.napari-nifti.dist</li>
    <li>python312Packages.netbox-documents</li>
    <li>python312Packages.netbox-documents.dist</li>
    <li>python312Packages.nibabel</li>
    <li>python312Packages.nibabel.dist</li>
    <li>python312Packages.nilearn</li>
    <li>python312Packages.nilearn.dist</li>
    <li>python312Packages.nipy</li>
    <li>python312Packages.nipy.dist</li>
    <li>python312Packages.nipype</li>
    <li>python312Packages.nipype.dist</li>
    <li>python312Packages.nitime</li>
    <li>python312Packages.nitime.dist</li>
    <li>python312Packages.nitransforms</li>
    <li>python312Packages.nitransforms.dist</li>
    <li>python312Packages.niworkflows</li>
    <li>python312Packages.niworkflows.dist</li>
    <li>python312Packages.odc-geo</li>
    <li>python312Packages.odc-geo.dist</li>
    <li>python312Packages.odc-loader</li>
    <li>python312Packages.odc-loader.dist</li>
    <li>python312Packages.odc-stac</li>
    <li>python312Packages.odc-stac.dist</li>
    <li>python312Packages.osmnx</li>
    <li>python312Packages.osmnx.dist</li>
    <li>python312Packages.pandera</li>
    <li>python312Packages.pandera.dist</li>
    <li>python312Packages.plotnine</li>
    <li>python312Packages.plotnine.dist</li>
    <li>python312Packages.pybids</li>
    <li>python312Packages.pybids.dist</li>
    <li>python312Packages.pydicom</li>
    <li>python312Packages.pydicom.dist</li>
    <li>python312Packages.pygmt</li>
    <li>python312Packages.pygmt.dist</li>
    <li>python312Packages.pykrige</li>
    <li>python312Packages.pykrige.dist</li>
    <li>python312Packages.pylibjpeg</li>
    <li>python312Packages.pylibjpeg-openjpeg</li>
    <li>python312Packages.pylibjpeg-openjpeg.dist</li>
    <li>python312Packages.pylibjpeg-rle</li>
    <li>python312Packages.pylibjpeg-rle.dist</li>
    <li>python312Packages.pylibjpeg.dist</li>
    <li>python312Packages.pymatgen</li>
    <li>python312Packages.pymatgen.dist</li>
    <li>python312Packages.pymedio</li>
    <li>python312Packages.pymedio.dist</li>
    <li>python312Packages.pynetdicom</li>
    <li>python312Packages.pynetdicom.dist</li>
    <li>python312Packages.pyogrio</li>
    <li>python312Packages.pyogrio.dist</li>
    <li>python312Packages.pyorthanc</li>
    <li>python312Packages.pyorthanc.dist</li>
    <li>python312Packages.pyotb</li>
    <li>python312Packages.pyotb.dist</li>
    <li>python312Packages.pyradiomics</li>
    <li>python312Packages.pyradiomics.dist</li>
    <li>python312Packages.python-csxcad</li>
    <li>python312Packages.python-csxcad.dist</li>
    <li>python312Packages.python-mapnik</li>
    <li>python312Packages.python-mapnik.dist</li>
    <li>python312Packages.python-openems</li>
    <li>python312Packages.python-openems.dist</li>
    <li>python312Packages.pyvista</li>
    <li>python312Packages.pyvista.dist</li>
    <li>python312Packages.rasterio</li>
    <li>python312Packages.rasterio.dist</li>
    <li>python312Packages.rio-stac</li>
    <li>python312Packages.rio-stac.dist</li>
    <li>python312Packages.rioxarray</li>
    <li>python312Packages.rioxarray.dist</li>
    <li>python312Packages.sfepy</li>
    <li>python312Packages.sfepy.dist</li>
    <li>python312Packages.shimmy</li>
    <li>python312Packages.shimmy.dist</li>
    <li>python312Packages.simpleitk</li>
    <li>python312Packages.simpleitk.dist</li>
    <li>python312Packages.spyder-kernels</li>
    <li>python312Packages.spyder-kernels.dist</li>
    <li>python312Packages.sumo</li>
    <li>python312Packages.sumo.dist</li>
    <li>python312Packages.templateflow</li>
    <li>python312Packages.templateflow.dist</li>
    <li>python312Packages.torchio</li>
    <li>python312Packages.torchio.dist</li>
    <li>python312Packages.trx-python</li>
    <li>python312Packages.trx-python.dist</li>
    <li>python312Packages.vtk</li>
    <li>python312Packages.wktutils</li>
    <li>python312Packages.wktutils.dist</li>
    <li>python312Packages.xnatpy</li>
    <li>python312Packages.xnatpy.dist</li>
    <li>python313Packages.basemap</li>
    <li>python313Packages.basemap.dist</li>
    <li>python313Packages.bsuite</li>
    <li>python313Packages.bsuite.dist</li>
    <li>python313Packages.cartopy</li>
    <li>python313Packages.cartopy.dist</li>
    <li>python313Packages.dcmstack</li>
    <li>python313Packages.dcmstack.dist</li>
    <li>python313Packages.deid</li>
    <li>python313Packages.deid.dist</li>
    <li>python313Packages.dicom-numpy</li>
    <li>python313Packages.dicom-numpy.dist</li>
    <li>python313Packages.dicom2nifti</li>
    <li>python313Packages.dicom2nifti.dist</li>
    <li>python313Packages.dicomweb-client</li>
    <li>python313Packages.dicomweb-client.dist</li>
    <li>python313Packages.dipy</li>
    <li>python313Packages.dipy.dist</li>
    <li>python313Packages.django-bootstrap4</li>
    <li>python313Packages.django-bootstrap4.dist</li>
    <li>python313Packages.django-bootstrap5</li>
    <li>python313Packages.django-bootstrap5.dist</li>
    <li>python313Packages.drf-extra-fields</li>
    <li>python313Packages.drf-extra-fields.dist</li>
    <li>python313Packages.f3d</li>
    <li>python313Packages.f3d.man</li>
    <li>python313Packages.fast-simplification</li>
    <li>python313Packages.fast-simplification.dist</li>
    <li>python313Packages.fiona</li>
    <li>python313Packages.fiona.dist</li>
    <li>python313Packages.folium</li>
    <li>python313Packages.folium.dist</li>
    <li>python313Packages.fslpy</li>
    <li>python313Packages.fslpy.dist</li>
    <li>python313Packages.gdcm</li>
    <li>python313Packages.geoarrow-pandas</li>
    <li>python313Packages.geoarrow-pandas.dist</li>
    <li>python313Packages.geoarrow-pyarrow</li>
    <li>python313Packages.geoarrow-pyarrow.dist</li>
    <li>python313Packages.geodatasets</li>
    <li>python313Packages.geodatasets.dist</li>
    <li>python313Packages.geopandas</li>
    <li>python313Packages.geopandas.dist</li>
    <li>python313Packages.geoparquet</li>
    <li>python313Packages.geoparquet.dist</li>
    <li>python313Packages.gstools</li>
    <li>python313Packages.gstools.dist</li>
    <li>python313Packages.heudiconv</li>
    <li>python313Packages.heudiconv.dist</li>
    <li>python313Packages.highdicom</li>
    <li>python313Packages.highdicom.dist</li>
    <li>python313Packages.inequality</li>
    <li>python313Packages.inequality.dist</li>
    <li>python313Packages.itk</li>
    <li>python313Packages.libpysal</li>
    <li>python313Packages.libpysal.dist</li>
    <li>python313Packages.lida</li>
    <li>python313Packages.lida.dist</li>
    <li>python313Packages.mapclassify</li>
    <li>python313Packages.mapclassify.dist</li>
    <li>python313Packages.medpy</li>
    <li>python313Packages.medpy.dist</li>
    <li>python313Packages.medvol</li>
    <li>python313Packages.medvol.dist</li>
    <li>python313Packages.momepy</li>
    <li>python313Packages.momepy.dist</li>
    <li>python313Packages.morecantile</li>
    <li>python313Packages.morecantile.dist</li>
    <li>python313Packages.msticpy</li>
    <li>python313Packages.msticpy.dist</li>
    <li>python313Packages.napari-nifti</li>
    <li>python313Packages.napari-nifti.dist</li>
    <li>python313Packages.netbox-documents</li>
    <li>python313Packages.netbox-documents.dist</li>
    <li>python313Packages.nibabel</li>
    <li>python313Packages.nibabel.dist</li>
    <li>python313Packages.nilearn</li>
    <li>python313Packages.nilearn.dist</li>
    <li>python313Packages.nipy</li>
    <li>python313Packages.nipy.dist</li>
    <li>python313Packages.nipype</li>
    <li>python313Packages.nipype.dist</li>
    <li>python313Packages.nitime</li>
    <li>python313Packages.nitime.dist</li>
    <li>python313Packages.nitransforms</li>
    <li>python313Packages.nitransforms.dist</li>
    <li>python313Packages.niworkflows</li>
    <li>python313Packages.niworkflows.dist</li>
    <li>python313Packages.odc-geo</li>
    <li>python313Packages.odc-geo.dist</li>
    <li>python313Packages.odc-loader</li>
    <li>python313Packages.odc-loader.dist</li>
    <li>python313Packages.odc-stac</li>
    <li>python313Packages.odc-stac.dist</li>
    <li>python313Packages.osmnx</li>
    <li>python313Packages.osmnx.dist</li>
    <li>python313Packages.pandera</li>
    <li>python313Packages.pandera.dist</li>
    <li>python313Packages.plotnine</li>
    <li>python313Packages.plotnine.dist</li>
    <li>python313Packages.pybids</li>
    <li>python313Packages.pybids.dist</li>
    <li>python313Packages.pydicom</li>
    <li>python313Packages.pydicom.dist</li>
    <li>python313Packages.pygmt</li>
    <li>python313Packages.pygmt.dist</li>
    <li>python313Packages.pykrige</li>
    <li>python313Packages.pykrige.dist</li>
    <li>python313Packages.pylibjpeg</li>
    <li>python313Packages.pylibjpeg-openjpeg</li>
    <li>python313Packages.pylibjpeg-openjpeg.dist</li>
    <li>python313Packages.pylibjpeg-rle</li>
    <li>python313Packages.pylibjpeg-rle.dist</li>
    <li>python313Packages.pylibjpeg.dist</li>
    <li>python313Packages.pymedio</li>
    <li>python313Packages.pymedio.dist</li>
    <li>python313Packages.pynetdicom</li>
    <li>python313Packages.pynetdicom.dist</li>
    <li>python313Packages.pyogrio</li>
    <li>python313Packages.pyogrio.dist</li>
    <li>python313Packages.pyorthanc</li>
    <li>python313Packages.pyorthanc.dist</li>
    <li>python313Packages.pyotb</li>
    <li>python313Packages.pyotb.dist</li>
    <li>python313Packages.pyradiomics</li>
    <li>python313Packages.pyradiomics.dist</li>
    <li>python313Packages.python-csxcad</li>
    <li>python313Packages.python-csxcad.dist</li>
    <li>python313Packages.python-mapnik</li>
    <li>python313Packages.python-mapnik.dist</li>
    <li>python313Packages.pyvista</li>
    <li>python313Packages.pyvista.dist</li>
    <li>python313Packages.rasterio</li>
    <li>python313Packages.rasterio.dist</li>
    <li>python313Packages.rio-stac</li>
    <li>python313Packages.rio-stac.dist</li>
    <li>python313Packages.rioxarray</li>
    <li>python313Packages.rioxarray.dist</li>
    <li>python313Packages.simpleitk</li>
    <li>python313Packages.simpleitk.dist</li>
    <li>python313Packages.spyder-kernels</li>
    <li>python313Packages.spyder-kernels.dist</li>
    <li>python313Packages.templateflow</li>
    <li>python313Packages.templateflow.dist</li>
    <li>python313Packages.torchio</li>
    <li>python313Packages.torchio.dist</li>
    <li>python313Packages.trx-python</li>
    <li>python313Packages.trx-python.dist</li>
    <li>python313Packages.vtk</li>
    <li>python313Packages.wktutils</li>
    <li>python313Packages.wktutils.dist</li>
    <li>python313Packages.xnatpy</li>
    <li>python313Packages.xnatpy.dist</li>
    <li>qcsxcad</li>
    <li>qcsxcad.dev</li>
    <li>qgis</li>
    <li>qgis-ltr</li>
    <li>qmapshack</li>
    <li>rtabmap</li>
    <li>saga</li>
    <li>simpleitk</li>
    <li>sumo</li>
    <li>supercell-wx</li>
    <li>survex</li>
    <li>therion</li>
    <li>tunnelx</li>
    <li>udig</li>
    <li>valhalla</li>
    <li>vpv</li>
    <li>vtk (vtk_9_5)</li>
    <li>vtk-dicom</li>
    <li>vtk-full</li>
    <li>vtkWithQt6</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 29 packages built:</summary>
  <ul>
    <li>gnudatalanguage</li>
    <li>libLAS</li>
    <li>libgeotiff</li>
    <li>libgeotiff.dev</li>
    <li>librasterlite2</li>
    <li>libspatialite</li>
    <li>libspatialite.dev</li>
    <li>nominatim</li>
    <li>nominatim.dist</li>
    <li>octavePackages.octproj</li>
    <li>osm2pgsql</li>
    <li>proj</li>
    <li>proj.dev</li>
    <li>python312Packages.labelbox</li>
    <li>python312Packages.labelbox.dist</li>
    <li>python312Packages.opensfm</li>
    <li>python312Packages.opensfm.dist</li>
    <li>python312Packages.pyproj</li>
    <li>python312Packages.pyproj.dist</li>
    <li>python313Packages.labelbox</li>
    <li>python313Packages.labelbox.dist</li>
    <li>python313Packages.opensfm</li>
    <li>python313Packages.opensfm.dist</li>
    <li>python313Packages.pyproj</li>
    <li>python313Packages.pyproj.dist</li>
    <li>spatialite-gui</li>
    <li>spatialite-tools</li>
    <li>xastir</li>
    <li>xygrib</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>gdal</summary>
<pre>    
&gt;       assert (
            wkt
            == &quot;&quot;&quot;COMPD_CS[&quot;TestMS&quot;,GEOGCS[&quot;NAD83(2011)&quot;,DATUM[&quot;NAD83_National_Spatial_Reference_System_2011&quot;,SPHEROID[&quot;GRS 1980&quot;,6378137,298.257222101004,AUTHORITY[&quot;EPSG&quot;,&quot;7019&quot;]],AUTHORITY[&quot;EPSG&quot;,&quot;1116&quot;]],PRIMEM[&quot;Greenwich&quot;,0],UNIT[&quot;degree&quot;,0.0174532925199433,AUTHORITY[&quot;EPSG&quot;,&quot;9122&quot;]],AXIS[&quot;Latitude&quot;,NORTH],AXIS[&quot;Longitude&quot;,EAST],AUTHORITY[&quot;EPSG&quot;,&quot;6318&quot;]],VERT_CS[&quot;NAVD88 height&quot;,VERT_DATUM[&quot;North American Vertical Datum 1988&quot;,2005,AUTHORITY[&quot;EPSG&quot;,&quot;5103&quot;]],UNIT[&quot;metre&quot;,1,AUTHORITY[&quot;EPSG&quot;,&quot;9001&quot;]],AXIS[&quot;Up&quot;,UP]]]&quot;&quot;&quot;
        )
E       assert &#x27;COMPD_CS[&quot;Te...IS[&quot;Up&quot;,UP]]]&#x27; == &#x27;COMPD_CS[&quot;Te...IS[&quot;Up&quot;,UP]]]&#x27;
E         
E         Skipping 368 identical leading characters in diff, use -v to show
E         - RT_DATUM[&quot;North American Vertical Datum 1988&quot;,2005,AUTHORITY[&quot;EPSG&quot;,&quot;5103&quot;]],UNIT[&quot;metre&quot;,1,AUTHORITY[&quot;EPSG&quot;,&quot;9001&quot;]],AXIS[&quot;Up&quot;,UP]]]
E         + RT_DATUM[&quot;Derived California Orthometric Heights of 1988 epoch 2025&quot;,2005],UNIT[&quot;metre&quot;,1,AUTHORITY[&quot;EPSG&quot;,&quot;9001&quot;]],AXIS[&quot;Up&quot;,UP]]]

/build/source/build/autotest/gcore/tiff_srs.py:1295: AssertionError
=========================== short test summary info ============================
FAILED gcore/tiff_srs.py::test_tiff_srs_read_compound_with_VerticalCitationGeoKey_only - assert &#x27;COMPD_CS[&quot;Te...IS[&quot;Up&quot;,UP]]]&#x27; == &#x27;COMPD_CS[&quot;Te...IS[&quot;Up&quot;,UP]]]&#x27;
  
  Skipping 368 identical leading characters in diff, use -v to show
  - RT_DATUM[&quot;North American Vertical Datum 1988&quot;,2005,AUTHORITY[&quot;EPSG&quot;,&quot;5103&quot;]],UNIT[&quot;metre&quot;,1,AUTHORITY[&quot;EPSG&quot;,&quot;9001&quot;]],AXIS[&quot;Up&quot;,UP]]]
  + RT_DATUM[&quot;Derived California Orthometric Heights of 1988 epoch 2025&quot;,2005],UNIT[&quot;metre&quot;,1,AUTHORITY[&quot;EPSG&quot;,&quot;9001&quot;]],AXIS[&quot;Up&quot;,UP]]]
= 1 failed, 16029 passed, 1583 skipped, 37 deselected, 5 xfailed in 418.09s (0:06:58) =
100 - done.</pre>
</details>
<details>
<summary>gdalMinimal</summary>
<pre>    
&gt;       assert (
            wkt
            == &quot;&quot;&quot;COMPD_CS[&quot;TestMS&quot;,GEOGCS[&quot;NAD83(2011)&quot;,DATUM[&quot;NAD83_National_Spatial_Reference_System_2011&quot;,SPHEROID[&quot;GRS 1980&quot;,6378137,298.257222101004,AUTHORITY[&quot;EPSG&quot;,&quot;7019&quot;]],AUTHORITY[&quot;EPSG&quot;,&quot;1116&quot;]],PRIMEM[&quot;Greenwich&quot;,0],UNIT[&quot;degree&quot;,0.0174532925199433,AUTHORITY[&quot;EPSG&quot;,&quot;9122&quot;]],AXIS[&quot;Latitude&quot;,NORTH],AXIS[&quot;Longitude&quot;,EAST],AUTHORITY[&quot;EPSG&quot;,&quot;6318&quot;]],VERT_CS[&quot;NAVD88 height&quot;,VERT_DATUM[&quot;North American Vertical Datum 1988&quot;,2005,AUTHORITY[&quot;EPSG&quot;,&quot;5103&quot;]],UNIT[&quot;metre&quot;,1,AUTHORITY[&quot;EPSG&quot;,&quot;9001&quot;]],AXIS[&quot;Up&quot;,UP]]]&quot;&quot;&quot;
        )
E       assert &#x27;COMPD_CS[&quot;Te...IS[&quot;Up&quot;,UP]]]&#x27; == &#x27;COMPD_CS[&quot;Te...IS[&quot;Up&quot;,UP]]]&#x27;
E         
E         Skipping 368 identical leading characters in diff, use -v to show
E         - RT_DATUM[&quot;North American Vertical Datum 1988&quot;,2005,AUTHORITY[&quot;EPSG&quot;,&quot;5103&quot;]],UNIT[&quot;metre&quot;,1,AUTHORITY[&quot;EPSG&quot;,&quot;9001&quot;]],AXIS[&quot;Up&quot;,UP]]]
E         + RT_DATUM[&quot;Derived California Orthometric Heights of 1988 epoch 2025&quot;,2005],UNIT[&quot;metre&quot;,1,AUTHORITY[&quot;EPSG&quot;,&quot;9001&quot;]],AXIS[&quot;Up&quot;,UP]]]

/build/source/build/autotest/gcore/tiff_srs.py:1295: AssertionError
=========================== short test summary info ============================
FAILED gcore/tiff_srs.py::test_tiff_srs_read_compound_with_VerticalCitationGeoKey_only - assert &#x27;COMPD_CS[&quot;Te...IS[&quot;Up&quot;,UP]]]&#x27; == &#x27;COMPD_CS[&quot;Te...IS[&quot;Up&quot;,UP]]]&#x27;
  
  Skipping 368 identical leading characters in diff, use -v to show
  - RT_DATUM[&quot;North American Vertical Datum 1988&quot;,2005,AUTHORITY[&quot;EPSG&quot;,&quot;5103&quot;]],UNIT[&quot;metre&quot;,1,AUTHORITY[&quot;EPSG&quot;,&quot;9001&quot;]],AXIS[&quot;Up&quot;,UP]]]
  + RT_DATUM[&quot;Derived California Orthometric Heights of 1988 epoch 2025&quot;,2005],UNIT[&quot;metre&quot;,1,AUTHORITY[&quot;EPSG&quot;,&quot;9001&quot;]],AXIS[&quot;Up&quot;,UP]]]
= 1 failed, 14314 passed, 3118 skipped, 42 deselected, 5 xfailed in 220.85s (0:03:40) =
100 - done.</pre>
</details>
<details>
<summary>python312Packages.gdal</summary>
<pre>    
&gt;       assert (
            wkt
            == &quot;&quot;&quot;COMPD_CS[&quot;TestMS&quot;,GEOGCS[&quot;NAD83(2011)&quot;,DATUM[&quot;NAD83_National_Spatial_Reference_System_2011&quot;,SPHEROID[&quot;GRS 1980&quot;,6378137,298.257222101004,AUTHORITY[&quot;EPSG&quot;,&quot;7019&quot;]],AUTHORITY[&quot;EPSG&quot;,&quot;1116&quot;]],PRIMEM[&quot;Greenwich&quot;,0],UNIT[&quot;degree&quot;,0.0174532925199433,AUTHORITY[&quot;EPSG&quot;,&quot;9122&quot;]],AXIS[&quot;Latitude&quot;,NORTH],AXIS[&quot;Longitude&quot;,EAST],AUTHORITY[&quot;EPSG&quot;,&quot;6318&quot;]],VERT_CS[&quot;NAVD88 height&quot;,VERT_DATUM[&quot;North American Vertical Datum 1988&quot;,2005,AUTHORITY[&quot;EPSG&quot;,&quot;5103&quot;]],UNIT[&quot;metre&quot;,1,AUTHORITY[&quot;EPSG&quot;,&quot;9001&quot;]],AXIS[&quot;Up&quot;,UP]]]&quot;&quot;&quot;
        )
E       assert &#x27;COMPD_CS[&quot;Te...IS[&quot;Up&quot;,UP]]]&#x27; == &#x27;COMPD_CS[&quot;Te...IS[&quot;Up&quot;,UP]]]&#x27;
E         
E         Skipping 368 identical leading characters in diff, use -v to show
E         - RT_DATUM[&quot;North American Vertical Datum 1988&quot;,2005,AUTHORITY[&quot;EPSG&quot;,&quot;5103&quot;]],UNIT[&quot;metre&quot;,1,AUTHORITY[&quot;EPSG&quot;,&quot;9001&quot;]],AXIS[&quot;Up&quot;,UP]]]
E         + RT_DATUM[&quot;Derived California Orthometric Heights of 1988 epoch 2025&quot;,2005],UNIT[&quot;metre&quot;,1,AUTHORITY[&quot;EPSG&quot;,&quot;9001&quot;]],AXIS[&quot;Up&quot;,UP]]]

/build/source/build/autotest/gcore/tiff_srs.py:1295: AssertionError
=========================== short test summary info ============================
FAILED gcore/tiff_srs.py::test_tiff_srs_read_compound_with_VerticalCitationGeoKey_only - assert &#x27;COMPD_CS[&quot;Te...IS[&quot;Up&quot;,UP]]]&#x27; == &#x27;COMPD_CS[&quot;Te...IS[&quot;Up&quot;,UP]]]&#x27;
  
  Skipping 368 identical leading characters in diff, use -v to show
  - RT_DATUM[&quot;North American Vertical Datum 1988&quot;,2005,AUTHORITY[&quot;EPSG&quot;,&quot;5103&quot;]],UNIT[&quot;metre&quot;,1,AUTHORITY[&quot;EPSG&quot;,&quot;9001&quot;]],AXIS[&quot;Up&quot;,UP]]]
  + RT_DATUM[&quot;Derived California Orthometric Heights of 1988 epoch 2025&quot;,2005],UNIT[&quot;metre&quot;,1,AUTHORITY[&quot;EPSG&quot;,&quot;9001&quot;]],AXIS[&quot;Up&quot;,UP]]]
= 1 failed, 16029 passed, 1583 skipped, 37 deselected, 5 xfailed in 409.60s (0:06:49) =
100 - done.</pre>
</details>
</details>

--- a/pkgs/by-name/pr/proj/package.nix
+++ b/pkgs/by-name/pr/proj/package.nix
@@ -17,13 +17,13 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "proj";
-  version = "9.6.1";
+  version = "9.7.0";
 
   src = fetchFromGitHub {
     owner = "OSGeo";
     repo = "PROJ";
     rev = finalAttrs.version;
-    hash = "sha256-81wrwBB11SKhq2dTBrvbuUd97iYkTYYrYyKpk2IlHAA=";
+    hash = "sha256-Vdznj9WGuws1p+owDNHlVERjOM3fS1+RBtqe01q500E=";
   };
 
   patches = [

